nest-forms-backend/src/forms/forms.service.ts

```diff
@@ -149,11 +149,7 @@ export default class FormsService {
       )
     }
 
-    const { hasAccess } = await this.formAccessService.checkAccessByInstance(
```

Comment:
This is already handled by the controller. The `getForm` is the only usage. On top of that, controller also allows migration access to the form and this check doesn't, therefore forms that need migration doesn't work now.

https://github.com/bratislava/konto.bratislava.sk/blob/e3c9f71bef4db5b61c6d04ddc14ddf8e68628776/nest-forms-backend/src/nases/nases.controller.ts#L67